CODEOWNERS

```diff
@@ -14,6 +14,7 @@
 /tools/     @costinm @ldemailly
 /pkg/       @istio/reviewers-istio-pkg
 /tests/     @istio/reviewers-istio-tests
+/istioctl/  @istio/reviewers-istio-istioctl
```

Comment:
This is a bit of a stutter, but consistent with other reviewer groups. `reviewer-istioctl` would also work since `istio` is already prefixed to the name.